AzureWebAPI/Controllers/IPConfigController.cs

```diff
@@ -18,13 +20,17 @@ public async Task<IActionResult> RunCommand()
         {
             runCommandOutput = await Azure.RunCommand("IPConfig");
 
+            _logger.LogInformation("IPConfig command initiated succesfully at {DT}", DateTime.UtcNow.ToLocalTime().ToLongTimeString());

```

Comment:
Use readable names for placeholders: {Timestamp}, {DateTime}, {UserName} and so on.
When we will start using structured logging you will see why it is good practice. FYI, https://github.com/serilog/serilog/wiki/Structured-Data

Reply:
You don't need to add a date and time to every message. The logging library usually implements it for us.